ipatests/azure/azure-run-tests.sh

```diff
@@ -33,6 +33,8 @@ if [ "$install_result" -eq 0 ] ; then
 	echo "Installation complete. Performance of individual steps:"
 	grep 'service duration:' /var/log/ipaserver-install.log | sed -e 's/DEBUG //g'
 
+        export PATH=/root/.local/bin:$PATH
```

Comment:
Please use tab instead of space here.

Reply:
yes, sure. I saw this and going to fix on rebase (due to `test_smb`)